app/config.go

```diff
@@ -29,9 +29,17 @@ type Config struct {
 }
 
 type IMConfig struct {
-	GCP *GCPIMConfig
+	Type IMType
+	GCP  *GCPIMConfig
 }
 
+type IMType string
+
+const (
+	UnixIMType IMType = "unix"
+	GCEIMType  IMType = "GCE"
```

Comment:
should we use "GCP" to keep it consistent?

Reply:
done, changed the account manager type too